.github/workflows/dapr-e2e-kind.yml

```
@@ -80,7 +80,7 @@ jobs:
       id: skip_check
       uses: fkirc/skip-duplicate-actions@v3.4.0
       with:
-        paths_ignore: '["**.md", ".codecov.yaml"]'
+        paths_ignore: '["**.md", ".codecov.yaml", ".github/ISSUE_TEMPLATE/*"]'
```

Can you also add `.github/**/**.yml` ?

```suggestion
        paths_ignore: '["**.md", ".codecov.yaml", ".github/**/**.yml", ".github/ISSUE_TEMPLATE/*"]'
```

---

I agree that removing the `yml` path is not ideal.